forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/delegates/FormTextAnalysisDelegate.java

```diff
@@ -103,12 +107,8 @@ private String getAnalysisUrl(){
         return httpServiceInvoker.getProperties().getProperty("analysis.url")+"/sentiment";
     }
 
-    private String getFormUrl(DelegateExecution execution){
```

Comment:
leave this function as it was, it is creating confusion. Everywhere just revert only this change where you  removed functions